appsignals-tests/contract-tests/src/test/java/software/amazon/opentelemetry/appsignals/test/jdbc/JdbcContractTestBase.java

```diff
@@ -130,20 +131,28 @@ protected void assertSemanticConventionsSpanAttributes(
         .satisfiesOnlyOnce(
             rss -> {
               assertThat(rss.getSpan().getKind()).isEqualTo(SPAN_KIND_CLIENT);
-              assertThat(rss.getSpan().getName())
-                  .isEqualTo(String.format("%s %s.%s", dbOperation, dbName, dbSqlTable));
               assertThat(rss.getSpan().getStatus().getCode().toString()).isEqualTo(otelStatusCode);
               var attributesList = rss.getSpan().getAttributesList();
               assertSemanticConventionsAttributes(
-                  attributesList, dbSqlTable, dbSystem, dbOperation, dbUser, dbName, jdbcUrl);
+                  attributesList, dbSystem, dbUser, dbName, jdbcUrl);
+              assertSemanticConventionsAttributesForOperation(
+                  attributesList, dbOperation, dbSqlTable);
             });
   }
 
+  private void assertSemanticConventionForOperation(
```

Comment:
This isn't called anywhere...

Reply:
I have mentioned down below that I was testing locally :( 

After rebasing I didn't clean back my local history sorry for the inconvenience !